### ISSUE TITLE
Delta E should be an option under Options > Color Contrast Value

### ISSUE THREAD

**Issue by pawn002** [pawn002/color-pair-quick-iterator#130]
Delta E is offered in the Color Metadata section but there are likely users who will benefit from more ready access to Delta E similar to what the other options provide.
